cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -61,6 +61,7 @@ var (
 	retryIntervalStart   = flag.Duration("retry-interval-start", time.Second, "Initial retry interval of failed provisioning or deletion. It doubles with each failure, up to retry-interval-max.")
 	retryIntervalMax     = flag.Duration("retry-interval-max", 5*time.Minute, "Maximum retry interval of failed provisioning or deletion.")
 	workerThreads        = flag.Uint("worker-threads", 100, "Number of provisioner worker threads, in other words nr. of simultaneous CSI calls.")
+	finalizerThreads     = flag.Uint("cloning-protection-threads", 5, "Number of simultaniously running threads, handling cloning finalizer removal")
```

Comment:
I'm thinking even 1 may be sufficient as a default. Cloning shouldn't be that frequent?